.github/actions/e2e-atomic-angular/action.yml

```diff
@@ -16,9 +16,8 @@ runs:
       name: Run Cypress
       with:
         browser: chrome
-        command-prefix: 'npx cypress-repeat run -n 3 --until-passes --rerun-failed-only --'
-        working-directory: ./packages/samples/angular
-        start: npm run dev
+        command-prefix: 'npx -w @coveo/atomic-angular-samples cypress-repeat run -n 3 --until-passes --rerun-failed-only --'
```

Comment:
Would it be possible to extract common commands such as `cypress-repeat run -n 3 --until-passes --rerun-failed-only` to common nx config?

Reply:
Yes and no.

Technically, Nx doesn't allow us to create "default" tasks. However, we have an Nx plugin in our workspace (under `utils/nx-plugin`) which already generates Nx tasks, so it wouldn't be much more work to add a default `cypress-repeat` task.

I'll give it a shot.